lib/root.js

```diff
@@ -15,6 +15,14 @@ process.on('exit', function (code) {
     process.exit(1)
 })
 
+// need to autoend if a teardown is added.
+// otherwise we may never see process.on('exit')!
+tap.tearDown = tap.afterAll = function (fn) {
+  var ret = Test.prototype.afterAll.apply(this, arguments)
+  this.autoend()
```

Comment:
It seems odd to me that `tap.tearDown()` != `t.tearDown()`... if they have to be different (and I think that may be the case), does it even make sense to have a root level `tap.tearDown()`?